GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1800,7 +1810,6 @@ private void SetGitModule(object sender, GitModuleEventArgs e)
 
                     // If we're applying custom branch or revision filters - reset them
                     RevisionGrid.DisableFilters();
-                    SetPathFilter(pathFilter: null);
```

Comment:
This triggers a Grid refresh, so this must not occur (see also comments for the commit). 

Maybe the `CanRefresh` check in `RequestRefresh` should at least ``Debug.Assert`
Maybe add `RevisionGrid.DisableRefreshRevisions()`/`RevisionGrid.ResumeRefreshRevisions()` in `GitSetModule` to permanently avoid this as the refresh is done in `UICommands_PostRepositoryChanged()`

Reply:
This is likely because of the changes to `ResumeRefreshRevisions`